mint/mint.go

```diff
@@ -7,10 +7,10 @@ import (
 	"crypto/sha256"
 	"errors"
 	"fmt"
-
 	"github.com/lightninglabs/aperture/lsat"
 	"github.com/lightningnetwork/lnd/lntypes"
 	"gopkg.in/macaroon.v2"
+	"time"
```

Comment:
imports should be: first standard lib imports, new line, other imports. So `time` belongs in the standard section and the new line should stay